profiler/src/profiler/TracyView_FrameTimeline.cpp

```diff
@@ -95,7 +95,7 @@ void View::DrawTimelineFrames( const FrameData& frames )
 {
     const std::pair <int, int> zrange = m_worker.GetFrameRange( frames, m_vd.zvStart, m_vd.zvEnd );
     if( zrange.first < 0 ) return;
-    if( m_worker.GetFrameBegin( frames, zrange.first ) > m_vd.zvEnd || m_worker.GetFrameEnd( frames, zrange.second ) < m_vd.zvStart ) return;
```

Comment:
Here's additional moment: after checking for `frames.frames.empty()` I've faced crash when `GetFrameRange` returned the `frames.frames.end()` as a `zrange.second` and `m_worker.GetFrameEnd( frames, zrange.second )` went OOB